Jamulus.pro

```diff
@@ -1066,10 +1074,42 @@ contains(CONFIG, "opus_shared_lib") {
         DEFINES += USE_OPUS_SHARED_LIB
     }
 } else {
+    DEFINES += $$DEFINES_OPUS
     INCLUDEPATH += $$INCLUDEPATH_OPUS
     HEADERS += $$HEADERS_OPUS
     SOURCES += $$SOURCES_OPUS
     DISTFILES += $$DISTFILES_OPUS
+
+    contains(QT_ARCH, x86) | contains(QT_ARCH, x86_64) {
+        msvc {
+            SOURCES += $$SOURCES_OPUS_ARCH
```

Comment:
I understand this to mean that when building with MSVC, the sources are added as-is without any specific compiler flags. Can you add a clarifying comment why that is? Does MSVC figure them out automatically?

Reply:
`opus/win32/config.h` says `/* Can always compile SSE intrinsics (no special compiler flags necessary) */`

I think the the major difference is that gcc, given the flags, will use SSE instructions automatically (this happened to me a couple times before I worked out how to apply the flags to the correct files; I got illegal instruction crashes) even when no intrinsics are used.